src/sentry/api/bases/organization_events.py

```diff
@@ -108,3 +110,59 @@ def get_snuba_query_args_v2(self, request, organization, params):
             raise OrganizationEventsError(
                 'Boolean search operator OR and AND not allowed in this search.')
         return snuba_args
+
+    def next_event_id(self, request, organization, snuba_args, event):
```

Comment:
prev/next are basically the same function. Could we abstract these? (I feel like we did a similar function elsewhere before, not sure if that can be used here as well)

Reply:
Is this logic basically the same as the one in the SnubaEvent class ?
https://github.com/getsentry/sentry/blob/master/src/sentry/models/event.py#L600-L658

Reply:
Ya, it's pretty similar to that